components/tests.py

```diff
@@ -218,7 +218,7 @@ def setUpTestData(cls):
             title="Cool Component",
             description="Probably the coolest component you ever did see. It's magical.",
             catalog=Catalog.objects.get(id=cls.test_catalog.id),
-            controls=["ac-2.1", "ac-6.10", "ac-8", "au-6.1", "sc-2"],
+            controls=["ac-1", "ac-2", "at-1", "at-2", "at-3", "au-1", "au-2"],
```

Comment:
Was this because these controls didn't exist?

Reply:
Correct